jobs/ci-kubernetes-e2e-kubeadm-gce.sh

```diff
@@ -26,9 +26,18 @@ readonly testinfra="$(dirname "${0}")/.."
 
 export PROJECT="k8s-jkns-e2e-kubeadm-gce-ci"
 export KUBERNETES_PROVIDER=kubernetes-anywhere
+
+# This job only runs against the kubernetes repo, and bootstrap.py leaves the
+# current working directory at the repository root. Grab the SCM_REVISION so we
+# can use the .debs built during the bazel-build job that should have already
+# succeeded.
+export SCM_VERSION=$(./hack/print-workspace-status.sh | grep ^STABLE_BUILD_SCM_REVISION | cut -d' ' -f2)
```

Comment:
Wait, what? bootstrap.py should be launching an image, the runner should not be calling bootstrap.py